README.md

```diff
@@ -1,2 +1,16 @@
 # compass-circular-slider
-a circular slider that looks like a compass
+A customizable circular slider component written in TypeScript, designed to look and function like a compass. The component allows users to set an angle with an optional secondary slider for elevation.
+
+## Features
+
+- **Compass-Like Circular Slider**: Adjust the main angle with a draggable circular slider.
+- **Optional Secondary Slider**: Add an elevation slider around the main slider for an extra degree of customization.
```

Comment:
I really liked the readme!!

Make sure to update, since there are now 2 different components